tests/test_helper.tcl

```diff
@@ -186,12 +189,16 @@ proc srv {args} {
 }
 
 # Provide easy access to the client for the inner server. It's possible to
-# prepend the argument list with a negative level to access clients for
-# servers running in outer blocks.
+# prepend the argument list with a level to access clients for
+# servers running in outer blocks. The level can be provided as either a positive
+# or negative index.
 proc r {args} {
     set level 0
     if {[string is integer [lindex $args 0]]} {
         set level [lindex $args 0]
+        if {$level > 0} {
+            set level [expr -1*$level]
+        }
```

Comment:
isn't that wrong?
shouldn't we fix `proc srv` to support positive indexes (avoid the use of `end-` in that case).
i.e. negative index counts from the end, and positive one counts from the beginning.

Reply:
ohh, but then we have an issue with 0... didn't realize that till now.

i think that's an odd interface, and considering we changed all the lines from `R` to `r` maybe we can also use that opportunity to fix the sign.
or maybe we should revert that change and keep the capital `R`.
i'm not sure which is better...
i think i'm slightly leaning towards capital R and easier review of the history.

Reply:
Ok, based on the confusion, this is what I'm thinking as well. I'm going to keep the indentation, but revert the R changes.